pkg/pgmodel/metrics/database/metrics.go

```diff
@@ -397,6 +397,96 @@ var metrics = []metricQueryWrap{
 			timescaledb_information.jobs jobs
 				on jobs.job_id = stats.job_id
 			where jobs.proc_name = 'execute_maintenance_job' and stats.last_run_status = 'Failed'`,
+	}, {
+		metrics: append(gauges(
```

Comment:
Isn't it better to use histograms to represent `duration` so we can see the distribution and compute aggregates? I think overall we always use histograms to track durations

Reply:
Yeah, I remember it wasn't supported, and it isn't https://github.com/timescale/promscale/blob/master/pkg/pgmodel/metrics/database/database.go#L197. 

Would be nice to have in the future, but I don't consider it to be in the scope of this PR.

Reply:
1. I guess recording histograms in DB is more work.
2. Not sure I understand what you mean by longer-than-a-day interval? I assume jobs are continuously running in the background and we want to know p95 or avg. For me this is the same analogy as for tracking request duration.

Anyways if it's too much effort to store histograms in DB I am fine with using gauges for now . I just wanted to understand the reasons.

Reply:
1. It doesn't seem too hard to call `histogram.Observe`, AFAICT there's no need to bake a histogram on the DB side.
2. Jobs run once every 30m at most. Any sort of aggregation would only make sense on sufficiently long time intervals. Otherwise, you'd be looking at p95 or avg of a small number of values.

Reply:
In other words I'll try and play with histograms, but I don't think it should be a blocker.

Reply:
Well, I made them histograms 🎉

Reply:
Awesome!